docs/contributing/settings.md

```diff
@@ -68,6 +69,7 @@ The settings object has a single sub-object `testing` which contains the configu
 |||
 |--|--|
 |**enableWebTests**|Flag to toggle if tests are run against the live services or not. If this is set to false none of the other sections are required.|
+|**enableGroupTests**|Flag to toggle if tests are run against an ephemeral M365 group or not. If this is set to false no M365 group will be created for tests.|
```

Comment:
We need to remove these references too...

Reply:
Removed!